spec/lib/search_spec.rb

```diff
@@ -94,4 +94,21 @@
       end
     end
   end
+
+  context "custom_eager_load" do
+    it "includes custom tables" do
+      expect(Search.new("test").send(:posts_eager_loads, Post).includes_values).to eq([:user, :post_search_data, { topic: [:category] }])
```

Comment:
`#posts_eager_loads` is a public method so we don't have to use `send` here. Whether the method should be private or not though is another question to be answered another day.

Reply:
yeah nah, there are to `private` keywords, I will remove the second one

Reply:
O in this case, let us not test for a private method. Instead we can just execute a search and test whether the association has been eager loaded with `post.association(:topic).loaded`?